addons/account/models/account_move.py

```diff
@@ -3339,22 +3347,15 @@ def _reverse_moves(self, default_values_list=None, cancel=False):
 
         reverse_moves = self.env['account.move']
         for move, default_values in zip(self, default_values_list):
-            default_values.update({
-                'move_type': TYPE_REVERSE_MAP[move.move_type],
-                'reversed_entry_id': move.id,
-                'partner_id': move.partner_id.id,
-            })
+            move._prepare_reversal_values(default_values)
```

Comment:
This seems useless, you can already add the values by inheriting `_reverse_moves` and zipping the list with self